templates/all-services/all-services-en.html

```diff
@@ -0,0 +1,738 @@
+			<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/canadian-passports.html">Passports</a></li>
+			<li><a href="https://www.canada.ca/en/revenue-agency/services/forms-publications/forms/rc518.html">RC 518 Declaration of Tax Residence for Individuals</a></li>
+			<li><a href="https://www.canada.ca/en/employment-social-development/services/sin.html">Social insurance number (SIN)</a></li>
+			<li><a href="https://www.canada.ca/en/services/benefits/publicpensions.html">Public pensions (CPP, OAS, GIS)</a></li>
+			<li><a href="https://www.canada.ca/en/employment-social-development/services/my-account/find-pac.html">Personal Access Code (PAC) for My Service Canada Account</a></li>
+			<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/visit-canada/eta.html">Electronic travel authorization (eTA)</a></li>
+			<li><a href="https://www.canada.ca/en/employment-social-development/programs/ei/ei-list/ei-roe.html">Record of employment (ROE)</a></li>
+			<li><a href="https://www.canada.ca/en/health-canada/services/health-cards.html">Provincial and territorial health cards</a></li>
+		</ul>
+	</div>
+</section>
+
+<section class="pb-3">
+	<div class="container">
+		<h2 class="h3 mb-4">Most requested services</h2>
+		<ul class="list-unstyled colcount-md-3 small bold-content lst-spcd-2 fnt-hdng">
+			<li><a href="https://www.canada.ca/en/government/sign-in-online-account.html">Sign in to an account</a></li>
+			<li><a href="https://www.canada.ca/en/government/change-address.html">Change your address</a></li>
+			<li><a href="https://www.canada.ca/en/public-services-procurement/services/payments-to-from-government/direct-deposit.html">Set up or change your direct deposit</a></li>
+			<li><a href="https://www.canada.ca/en/government/grants-funding.html">Find grants and funding</a></li>
+			<li><a href="https://www.canada.ca/en/services/jobs/opportunities.html">Find a job</a></li>
+			<li><a href="https://apps.cra-arc.gc.ca/ebci/rhpd/beta/entry">Payroll Deductions Online Calculator</a></li>
+		</ul>
+	</div>
+</section>
+
+<section class="bg-light pb-3">
+	<div class="container">
+		<h2 class="h3 mb-4">Supporting you through life's transitions</h2>
+		<ul class="list-unstyled colcount-md-3 small bold-content lst-spcd-2 fnt-hdng">
+			<li><a href="https://www.canada.ca/en/services/death.html">What to do when someone dies</a></li>
+			<li><a href="https://www.canada.ca/en/services/retirement.html">Learn and plan for your retirement</a></li>
+		</ul>
+	</div>
+</section>
+
+<div class="container">
+	<h2 class="mt-4">Access services by topic</h2>
+
+	<details class="mt-4">
+		<summary>Immigration and citizenship</summary>
+		<section class="bg-info row pt-3 pb-2">
+			<div class="col-md-3">
+				<h3 class="mt-0"><span class="fas fa-exclamation-circle text-info mr-2" aria-hidden="true"></span>Information on</h3>
```

Comment:
Note: We should automate the addition of `aria-hidden=true` for all font-awesome. We should not have the author to use aria directly. Such fix should be applied in JS.

```suggestion
				<h3 class="mt-0"><span class="fas fa-exclamation-circle text-info mr-2"></span>Information on</h3>
```